### PR TITLE
Issue #1752 - Add check for local settings file

### PR DESCRIPTION
**Pull request by alexfinnarn**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 # Ignore configuration files that may contain sensitive information.
 settings.php
+settings.*.php
```
Comment (quicksketch) [inline .gitignore:6]: We could simplify this rule into simply `settings*.php`. Do you think that would do any harm?

Comment (alexfinnarn) [inline .gitignore:6]: Seems fine to me and simplification is always good.

Comment (Gormartsen) [inline .gitignore:6]: This one PR is related: backdrop/backdrop#1310  Issue backdrop/backdrop-issues#1297

**Comment by quicksketch** [comment]
This needs updating per https://github.com/backdrop/backdrop-issues/issues/1752#issuecomment-201665135.

**Comment by jenlampton** [comment]
Closed in favor of https://github.com/backdrop/backdrop/pull/1332

**Pull request closed without merging**